Add Snowflake Cortex PAT auth guidance for SSO/Okta users (#9368)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-ai/wizard-byok.md

You can configure a provider in one of the following ways:

- [**Terminal commands**:](#configure-in-the-terminal) Best for users who want to configure providers from the shell.

> - [**Interactive session**:](#configure-in-the-tui) Best for most users working in the <Constant name="wizard" /> TUI. Use the `/providers` slash command.

Reviewer: [Non-blocker] Maybe we can spell out TUI here since this is the first instance?

wizard providers enable PROVIDER_NAME
```

> Replacing `PROVIDER_NAME` with the name of a supported provider, such as `openai`, `anthropic`, `bedrock`, `azure`, `gemini`, or `snowflake`. Then follow the prompts to enter your credentials.

Reviewer: ```suggestion
Replace `PROVIDER_NAME` with the name of a supported provider, such as `openai`, `anthropic`, `bedrock`, `azure`, `gemini`, or `snowflake`. Then, follow the prompts to enter your credentials.
```

wizard debug models
```

The `wizard providers configure snowflake` command walks you through the following prompts:

```
Enable this provider? [Y/n]:
Models to enable [1]:
Snowflake account ID:
Snowflake API base override (optional):
Paste API key/token, or press enter to configure it later:
```

<SimpleTable>

| Prompt | What to enter |
|--------|---------------|
| **Snowflake account ID** | Your Snowflake account identifier (for example, `myorg-myaccount`) |
| **Snowflake API base override (optional)** | Leave blank — this is only needed for custom or private Snowflake endpoints |
| **Paste API key/token** | Your authentication token — refer to [Authentication options](#authentication-options) in the next section |

</SimpleTable>

#### Authentication options

> The key/token field accepts a regular API token or a Programmatic Access Token (PAT), depending on how your Snowflake account is set up. Both go in the same place &mdash; the **Paste API key/token** prompt in the terminal, or **Set key/token** (option 3) in the TUI.

Reviewer: ```suggestion
The key/token field accepts a regular API token or a Programmatic Access Token (PAT), depending on how your Snowflake account is configured. Both are entered in the same place &mdash; the **Paste API key/token** prompt in the terminal, or **Set key/token** (option 3) in the TUI.
```

1. In your Snowflake account, [generate your API or PAT token](https://docs.snowflake.com/en/user-guide/programmatic-access-tokens#label-pat-generate).

> 2. Select **Generate token** and copy the token value

Reviewer: ```suggestion
2. Select **Generate token** and copy the token value.
```
